policies/modules/ROOT/pages/policies-included-directory.adoc

```diff
@@ -31,7 +31,7 @@ endif::[]
 | xref:policies::policies-included-pingfederate-oauth-token-enforcement.adoc[PingFederate OAuth 2.0 Token Enforcement] ^|  ^| ✓ | Allows access only to authorized client applications
 | xref:policies::policies-included-rate-limiting.adoc[Rate Limiting] ^| ✓ ^| ✓ | Monitors access to an API by defining the maximum number of requests processed within a period of time
 | xref:policies::policies-included-rate-limiting-sla.adoc[Rate Limiting: SLA-based] ^| ✓ ^| ✓ | Monitors access to an API by defining the maximum number of requests processed within a timespan, based on SLAs
-| xref:policies::policies-included-schema-validation.adoc[Schema Validation (OAS3)] ^| ✓ ^|  | Validates incoming traffic against a supplied OAS3 schema
+| xref:policies::policies-included-schema-validation.adoc[OAS3 Schema Validation Policy] ^| ✓ ^|  | Validates incoming traffic against a supplied OAS3 schema
```

Comment:
```suggestion
| xref:policies::policies-included-schema-validation.adoc[OAS3 Schema Validation] ^| ✓ ^|  | Validates incoming traffic against a supplied OAS3 schema
```

Also, since the title now starts with "O", please place this line in the appropriate place in the table (it's all alphabetical)